authors/urls.py

```diff
@@ -13,12 +13,11 @@
     1. Import the include() function: from django.conf.urls import url, include
     2. Add a URL to urlpatterns:  url(r'^blog/', include('blog.urls'))
 """
-from django.conf.urls import include, url
+from django.urls import include, path
 from django.contrib import admin
 
 urlpatterns = [
-    url(r'^admin/', admin.site.urls),
-
-    url(r'^api/', include('authors.apps.authentication.urls', namespace='authentication')),
+    path('admin/', admin.site.urls),
+    path('api/', include('authors.apps.authentication.urls')),
```

Comment:
I suggest you add the 'authentication' namespace back

Reply:
@malep2007, I found this during my research. Please check out the following links:
- (https://docs.djangoproject.com/en/2.1/intro/tutorial03/)
- (https://stackoverflow.com/questions/48608894/specifying-a-namespace-in-include-without-providing-an-app-name/49115466)
- (https://www.youtube.com/watch?v=v1r1Qu7kaZI&amp=&t=3s)

I will, however, follow the recommendation and put the namespaces back since it achieves the same result. Thanks very much for the review.